eclass/distutils-r1.eclass

```diff
@@ -973,22 +973,27 @@ distutils-r1_python_compile() {
 
 		# TODO: workaround for a bug in installer; remove once we depend
 		# on a properly fixed version
```

Comment:
While here, please add a bug reference to this so we know when to remove it in future.

Reply:
Done.